### ISSUE TITLE
"Publishing Failed" Error

### ISSUE THREAD

**Issue by torres126** [WordPress/gutenberg#9844]
**Description**
Publishing a post via Gutenberg results in a red "Publishing Failed" error.

**To Reproduce**
1. Write a post
2. Click publish

**Temporary Solution**
1. Save the post as a draft (which can be done successfully)
2. Publish it via Calypso OR go to Posts page and use Quick Edit to switch Draft to Published (note: tags and categories are lost in this second method)

Apparently, clearing cache resolves this issue, but that's very inconvenient for publishing posts?

**Expected behavior**
The post should have published without issues at all. Even when checking my site, it had not published until I followed the temporary solution I mentioned.

**Screenshots**

https://prnt.sc/ktooif

**Desktop (please complete the following information):**
 - iOS
 - Chrome
 - Version 49

**Additional context**
- I'm using the Gutenberg version offered on WordPress.com
- This appears to be a common issue, it has been mentioned more than once in the WordPress.com forums today and last night. 
https://en.forums.wordpress.com/topic/my-content-wont-publish-an-error-message-shows-in-red-publishing-of-post-failed/

Thanks! :)

**Comment by torres126** [comment]
Hi there, as expected, when writing another post today I got the same issue. This time the post was titled "CP Rewritten: “Hopefully” a 2007 Party" and I had to go to Quick Edit then change the status to published, but once again, the tags & categories were lost.

I didn't spend long writing this at all, no more than the reasonable amount of time to write a post and find a picture to add to it. Definitely less than half an hour. I did record the error I got though.

**Comment by torres126** [comment]
Hmmm I'm trying to attach the image via GitHub but to no success. Uploaded it to my Media Library instead: https://clubpenguinmountains.files.wordpress.com/2018/09/sgdfsfgdsdfgsgdfgdgf.png

Copy and pasted (can't seem to get blockquotes to work either):

"s1.wp.com/:167 Uncaught (in promise) undefined(anonymous function) @ s1.wp.com/:167(anonymous function) @ s1.wp.com/:2i @ s1.wp.com/:2j.fireWith @ s1.wp.com/:2e.(anonymous function) @ s1.wp.com/:2receive @ s1.wp.com/:1424n.event.dispatch @ s1.wp.com/:3r.handle @ s1.wp.com/:3
public-api.wordpress.com/wp-admin/rest-proxy/#https://clubpenguinmountains.wordpress.com:1 XMLHttpRequest cannot load https://developer.wordpress.com/docs/api/. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://public-api.wordpress.com' is therefore not allowed access."

**Comment by torres126** [comment]
Hi,

I'll be completely honest, I've stopped using Gutenberg, it isn't for me, but I'm still happy to help where I can.

https://developer.wordpress.com/docs/api/ loads fine for me (sorry about not being able to embed a photo for some reason): https://clubpenguinmountains.files.wordpress.com/2018/10/hdgfghfgdhfghhgfd.png

I'm also not using a VPN when accessing WordPress.com. Thank you! :)

**Comment by leecollings** [comment]
I completely agree with this, Gutenberg (despite the updates since it was introduced), is still broken, and in some cases unusable.

It really needs to just be ditched and reverted back to pre-Gutenberg when things just worked.

This is absolutely insane that I cannot save or publish a post, for an unknown reason.
